docs/installation-guide.md

```diff
@@ -1,73 +1,78 @@
 
 Head over to the [Discord Developer Portal](https://discord.com/developers/applications) and press the "New Application" button in the top right corner.
 
-Give it a name and press create. Then, head to the bot tab on the left and click "Add Bot". Click "Yes, do it!" Once you have your bot, scroll down and make sure that the "PRESENCE INTENT", "SERVER MEMBERS INTENT", and "MESSAGE CONTENT INTENT" switches are all toggled to on. Once that's done, go to the OAuth2 tab on the left sidebar, scroll down to the checkboxes, and select the ones for "bot" and "applications.commands". Copy the URL, enter it into your browser, select the server you want to add your Lily to, and click authorize. If everything worked correctly, your Lily instance should appear in the members list of that server.
+Give your application a name and press create. Then, head to the bot tab on the left and click "Add Bot". Click "Yes, do it!" Once you have your bot, scroll down and make sure that the "PRESENCE INTENT", "SERVER MEMBERS INTENT", and "MESSAGE CONTENT INTENT" switches are all toggled to on. Once that's done, go to the OAuth2 tab on the left sidebar, scroll down to the checkboxes, and select the ones for "bot" and "applications.commands". Copy the URL, enter it into your browser, select the server(s) you want to add your Lily to, and click authorize. If everything worked correctly, your Lily instance should appear in the members list of that server.
 
-### Step 2 - Clone the repository
+### Step 3 - Clone the repository
 
-To clone the Lily repository, open the command line for your respective OS and run `git clone https://github.com/IrisShaders/LilyBot.git`
-This will clone Lily to your home directory.
+To clone the Lily repository, open the command line for your respective OS and run `git clone https://github.com/IrisShaders/LilyBot.git` This will clone Lily to your home directory.
 
-### Step 3 - Set up your .env file
+### Step 4 - Set up your .env file
 
-In the root directory of the files you've cloned (It has files such as `README.md` and `CONTRIBUTING.md`), create a file named .env. You should fill your file using the format below with the relevant details filled in.
+In the root directory of the files you've cloned, create a file named .env. You should fill your file using the format below with the relevant details filled in.
 
 ```
 TOKEN=BOT_TOKEN_YEEF
-ADMIN_ROLE=ADMIN_ROLE_ID
-MODERATOR_ROLE=MODERATOR_ROLE_ID
-TRIAL_MODERATOR_ROLE=TRIAL_MODERATORS_ROLE_ID
-MODERATOR_PING_ROLE=PING_MODERATORS_ROLE_ID
-SUPPORT_ROLE=SUPPORT_TEAM_ROLE_ID
-GUILD_ID=GUILD_ID
-MOD_ACTION_LOG=MOD_ACTION_LOG_CHANNEL_ID
-MESSAGE_LOGS=MESSAGE_LOGS_CHANNEL_ID
-CONFIG_PATH=lily.toml
-JOIN_CHANNEL=JOIN_CHANNEL_ID
-SUPPORT_CHANNEL=SUPPORT_CHANNEL_ID
-GITHUB_OAUTH=GITHUB_OAUTH_TOKEN
+CUSTOM_COMMANDS_PATH=commands.toml
+TEST_GUILD_ID=GUILD_ID
+TEST_GUILD_CHANNEL=CHANNEL_ID
+GITHUB_OAUTH=GITHUB_OAUTH_TOKEN or null if you don't want GitHub commands
```

Comment:
There is actually another `.env` value that isn't required. `JDBC_URL` which needs to be given the value of `jdbc:sqlite:<path to where the database should go>`. It actually has a default value of `./data` folder in the root directory though, so it is absoloutely not required